scripts/jest/config.json

```diff
@@ -19,6 +19,7 @@
   },
   "setupFiles": [
     "<rootDir>/scripts/jest/setup/enzyme.js",
+    "<rootDir>/scripts/jest/setup/html_element.js",
```

Comment:
Jest uses jsdom which does not yet support the DOM function insertAdjacentElement. More information can be found at https://github.com/jsdom/jsdom/issues/1890

Reply:
Out of curiosity, can this be done with a polyfill? Or does it need to be done like this because of Jest's infrastructure?

Reply:
I haven't found an official polyfill for this, it's usually custom written, so this is supposed to be a polyfill, it's just in the `setup` folder, not `polyfills`